web/server/routers/post.ts

```diff
@@ -109,6 +121,67 @@ export const postRouter = router({
       });
     }
   }),
+  offer: procedure
+    .input(
+      z.object({
+        email: z.string(),
+        postOid: zodOidType,
+    })
+  ).mutation(async({input}) => {
+    const user = await findUserByEmail(input.email);
+    if (!user) {
+      throw new TRPCError({
+        message: "No user with given email exists.",
+        code: "BAD_REQUEST",
+      });
+    }
+    let email = "";
+    try {
+      const post = await getPost(input.postOid);
+      switch(post.type) {
+        case "fosterMove":
+          email = "foster@angelsrescue.org";
+          break;
+        case "return":
+          email = "returns@angelsrescue.org, foster@angelsrescue.org";
+          break;
+        case "temporary":
+          email = "tempfoster@angelsrescue.org";
+          break;
+        case "boarding":
+          email = "boardingadmin@angelsrescue.org"
+        case "shelter":
+        case "ownerSurrender":
+          email = "fosteroffer@angelsrescue.org";
+          break;
+      }
```

Comment:
Do the post type -> email mapping using an object with the `FosterType` enums as keys:
```js
{
  [FosterType.Return]: ["email here"],
  [FosterType.Temporary]: ["email here"],
  ...
}
```

You can either join the array of emails into a comma delimited string or just pass it into the `to` field for the transporter directly.